src/seedu/addressbook/data/person/Unit.java

```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Created by Herman on 27/6/2017.
+ */
+public class Unit {
```

Comment:
Missing header comment. All non-trivial classes and methods should have javadoc format header comments.